logstash-core/lib/logstash/runner.rb

```diff
@@ -19,9 +18,10 @@
 require "logstash/version"
 
 class LogStash::Runner < Clamp::StrictCommand
+  include LogStash::Util::Loggable
```

Comment:
With this include, given that `Loggable` module creates a `Logger` instance, it implies a log4j initialization before log4j config file is defined.
I suggest to not include Loggable here but after log4j config file has been precised (still in runner.rb).